{"name": "test_10", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 4", "trace": "@allure.feature(\"模块三\")\n    @allure.severity(\"trivial\")\n    def test_10():\n>       assert 3 == 4\nE       assert 3 == 4\n\ntest_1.py:76: AssertionError"}, "start": 1566181491957, "stop": 1566181491958, "uuid": "bd37424f-d091-447d-b061-93ea49a67cf0", "historyId": "cf3d2d54d7d03bd0942f84260d5454c3", "fullName": "test_1#test_10", "labels": [{"name": "severity", "value": "trivial"}, {"name": "feature", "value": "模块三"}, {"name": "parentSuite"}, {"name": "suite", "value": "test_1"}, {"name": "subSuite"}, {"name": "host", "value": "pc-guohonggang"}, {"name": "thread", "value": "9880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_1"}]}